tests/components/grid.test.ts

```diff
@@ -1443,6 +1443,27 @@ describe("Copy paste keyboard shortcut", () => {
     expect(getCellContent(model, "A2")).toEqual("things");
   });
 
+  test("Copied group of cells have correct 'text/html'", async () => {
+    setCellContent(model, "A1", "1");
+    setCellContent(model, "B2", "2");
+    setCellContent(model, "A2", "3");
+    setSelection(model, ["A1:B2"]);
+    document.activeElement!.dispatchEvent(
+      new KeyboardEvent("keydown", { key: "=", altKey: true, bubbles: true })
+    );
+    document.body.dispatchEvent(await getClipboardEvent("copy"));
+    expect(model.getters.getClipboardContent()["text/html"]).toEqual(
+      "<table><tr><td>1</td><td></td></tr><tr><td>3</td><td>2</td></tr></table>"
+    );
+  });
+
+  test("Copied single cells are not in a html table", async () => {
```

Comment:
These tests should be in `clipboard.test.ts`, and in the grid tests you should check if the clipboard given to the clipboardEvent has beed updated by grid.